apps/cli/src/credentials-storage.ts

```diff
@@ -7,14 +7,33 @@ export enum CredentialsStorageType {
   ACCOUNT = "user-account",
 }
 
+/**
+ * Find the filenames of all files in the `codemodDirectoryPath` that are for
+ * a given service. If the directory does not exist, silently return no results.
+ * @param service the service name to look for
+ * @returns a string array of all filenames that match the service name
+ */
+async function findServiceCredentialFiles(service: string): Promise<string[]> {
+  try {
+    return await readdir(codemodDirectoryPath).then((dir) =>
+      dir.filter((file) => file.startsWith(`${service}:`)),
+    );
+  } catch (error) {
+    // If the directory does not exist, treat it as "no results"
+    if ((error as NodeJS.ErrnoException).code === "ENOENT") {
```

Comment:
I also need to test what happens here if `readdir` throws an error that is not a `NodeJS.ErrnoException` or does not have a `.code` attribute